flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorBaseTest.java

```diff
@@ -220,7 +225,7 @@ private static Class<? extends Pojo> annotatePojoWithTable(String keyspace, Stri
     }
 
     @NotNull
-    private static Table createTableAnnotation(String keyspace, String tableName) {
+    private Table createTableAnnotation(String keyspace, String tableName) {
```

Comment:
Why is this no longer static?

Reply:
Because it is called only from `annotatePojoWithTable` method and this method was called only from the test methods so it should not have been static in the first place. So I'm cleaning this

Reply:
Why shouldn't it have been static in the first place? The general rule is "if it can be static, make it static.".

Reply:
ok, :+1: